sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -35,15 +35,37 @@ az acr create --name MyContainerRegistry --resource-group MyResourceGroup --loca
 
 ### Authenticate the client
 
-The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.
+The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.  You can set the environment variables `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, and `AZURE_CLIENT_SECRET` to configure `DefaultAzureCredential` to authenticate with AAD. For more information refer to [Azure Identity environment variables](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#environment-variables) or [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#defaultazurecredential).
```

Comment:
I believe only `EnvironmentCredential` will use those environment variables, so depending on the current execution environment or the options passed to DAC, these may be ignored.

Reply:
Isn't EnvironmentCredential the first the the DAC chain? Are there cases where this could be bypassed?

Reply:
I'm trying to push our readme's away from documenting using environment variables with the DAC as this requires setting up a service principal. Instead I think service client readme's should document that users can log in through the CLII and link to the Identity readme for more ways to authenticate your dev environment when using the DAC.

Reply:
> Isn't EnvironmentCredential the first the the DAC chain? Are there cases where this could be bypassed?

Yes - https://github.com/Azure/azure-sdk-for-net/blob/3065eaa87b8f068f1980fb8d9624678d7e5179e9/sdk/identity/Azure.Identity/src/DefaultAzureCredentialOptions.cs#L63

Reply:
@schaabs can you point me to an example of a service client README that does this the way you'd like to see it, so I can copy it?

Reply:
Updated this to reflect Scott's feedback